components/eam/src/chemistry/yaml/mo_setinv/setinv_output_ts_1415.py

```diff
@@ -0,0 +1,36 @@
+# Data at [lat=   39.0553303768561      , lon=   262.904774388088      , k=          30 , time step=        1415 ] 
+# This file was generated by E3SM.
+
+from math import nan as nan, inf as inf
+
+# Object is just a dynamic container that stores input/output data.
+class Object(object):
+    pass
+# Settings are stored here.
+settings = Object()
+# Input is stored here.
+input = Object()
+input.tfld=[[  0.21505833215545329E+003],]
+input.h2ovmr=[[  0.40263576550931807E-005],]
+input.vmr=[[  0.46650558221438391E-006,  0.22546191880225664E-011,  0.20523118193835984E-014,  0.83281009208800878E-012,  0.57132751197330756E-015,  0.93377758951275848E-015,  0.99230451450106709E-010,  0.27053840446118629E-009,  0.12992029652234340E-008,  0.40435083363759340E-010,  0.32241629640611341E-011,  0.10465006888940500E-010,  0.31547031277251843E-015,  0.28428952741242332E+010,  0.12822850635062660E-011,  0.19135183439573755E-010,  0.16454289552460293E-014,  0.28249178985865993E-019,  0.23960149416319237E+010,  0.12175048026332074E-012,  0.43627443941535684E-011,  0.51598376328203104E-013,  0.27726763402288446E-013,  0.18405761648998870E-012,  0.70699284694566954E-012,  0.59982905898811320E-018,  0.53258686376162419E+005,  0.24617446887877327E-010,  0.37350920924871430E-011,  0.27757720563517194E-020,  0.22564352129756331E+009,],]
+input.pmid=[[  0.13368217993210610E+005],]
+input.ncol=[[         4],]
+input.lchnk=[[       137],]
+input.pcols=[[         4],]
+input.pver=[[        72],]
+input.gas_pcnst=[[        31],]
+input.nfs=[[         8],]
+input.boltz_cgs=[[  0.13806500000000001E-015],]
+input.num_tracer_cnst=[[         4],]
+input.has_n2=[[         1],]
+input.m_ndx=[[         1],]
+input.n2_ndx=[[         2],]
+input.has_o2=[[         1],]
+input.o2_ndx=[[         3],]
+input.has_h2o=[[         1],]
+input.h2o_ndx=[[         4],]
+input.tracer_cnst_flds=[[cnst_O3,OH,NO3,HO2,],]
```

Comment:
These names need to be quoted so they're Python strings.

Reply:
Unfortunately, though, Skywalker won't be able to use string inputs, so we'll likely have to hardwire these species into our related tests.

Reply:
Sure, that's probably best. I'll make a note of the species list.

Reply:
As long as strings with quotes in them do no harm, I've gone ahead and inserted quotes for the string fields in a new batch of files.  If they do harm things though then I will go ahead and just remove the field.